appengine-java8/pubsub/src/main/java/com/example/appengine/pubsub/PubSubHome.java

```diff
@@ -45,5 +65,15 @@ private static String convertToHtmlTable(List<Message> messages) {
     return sb.toString();
   }
 
-  private PubSubHome() { }
+  private static String convertStringsToHtmlTable(List<String> strings) {
```

Comment:
I'm ok w/ this, but it really should be template based or anti-template based. https://github.com/agentgt/jatl

Reply:
done